docs/swagger.json

```diff
@@ -143,6 +143,49 @@
             }
         },
         "/api/v1/metadata/": {
+            "put": {
+                "tags": [
+                    "Metadata"
+                ],
+                "summary": "Force a new version of online metadata role(s).",
+                "description": "Force a new version of online metadata role(s). If the roles list is empty all roles will be updated. The new metadata version(s) will have extended expiration which will equal to: today + ROLE_NAME_EXPIRATION number of days, where ROLE_NAME_EXPIRATION is a tuf repository setting. Note: depending on which metadata role you want to update other online roles will likely be updated as well otherwise consistency will be lost.",
```

Comment:
What will happen if `datetime.now() + <ROLE_NAME_EXPIRATION number of days>` is less than the current expiration? I think we should either make it impossible or define it more strictly here (like extended or redused - practically it can also be the same, so we can just say "updated").

Reply:
What I mean is that this description creates expectations that don't strictly match the functionality.

Reply:
The whole idea is to be able to redefine current expiration.
For context: the idea of this feature came to life after I implemented `PUT /api/v1/config`.
With that API endpoint, you can change the expiration policy for each role.
Still, this policy will be enforced when an online role expires and the automatic job to bump that role is executed.
See: https://github.com/repository-service-tuf/repository-service-tuf-worker/blob/486e98cdbc16dd41c62c0ec8b473e2f94312ee9c/app.py#L139

The idea behind this new endpoint is that after you have called `PUT /api/v1/config` you can call this new endpoint `PUT /api/v1/metadata` and request a new metadata version of part or all online metadata roles.

For the future I have thought about adding a flag `force_new_version` to the `PUT /api/v1/config`  endpoint which will do the same think as this new endpoint but for all roles.

Reply:
As suggested by Ivana I added a commit that makes sure we cannot push negative expiration days.
Here is the commit: https://github.com/repository-service-tuf/repository-service-tuf-api/pull/513/commits/8a00f5bb795d3b73dc3313924c019f543302d2ef